packages/compiler-cli/src/ngtsc/diagnostics/src/error_code.ts

```diff
@@ -161,6 +161,15 @@ export enum ErrorCode {
   INJECTABLE_DUPLICATE_PROV = 9001,
 }
 
+/**
+ * @internal
+ */
+export const HAS_GUIDE = new Set([
```

Comment:
@twerske for some reasons I can't reply to your previous comment about "no-toplevel-property-access" tslint rule, so I'll post a reply here. Since the `RuntimeErrorCode` is a const enum, its values are inlined in the generated code ([see an example here](https://www.typescriptlang.org/play?#code/MYewdgzgLgBApmArgWxgJUWKBLZcCiAToSIQMIgAmcMA3gFAwwD0zMZAFgIZgDmNAEThQ4wHOBhEShCI0kANAApp8AZVUBJAPIA5APpkAEgEEdAcXwC9xgGIAVfGgOH8ZANKWYAXhgByAIwADIG+ADT0cqwwQgAOCNRgwACeMBpgAFai4mCSxKSyTGQAmmQAMhpkegIaVfiK+DoCDcXefgBMwWFyyloAahpNTjpadno2WgCqja2+Hf5dAL4RoJCwcMStGFi4BHnkVHAAdMVlFVU1TfWNzUUA3EA)), so the lint rule can be safely ignored, i.e. you can disable that tslint rule for the entire `HAS_GUIDE` block.

```suggestion
/* tslint:disable:no-toplevel-property-access */
export const HAS_GUIDE = new Set([ ... ]);
/* tslint:enable:no-toplevel-property-access */
```